lineapy/plugins/pipeline_writers.py

```diff
@@ -109,6 +114,46 @@ def _write_requirements(self) -> None:
         file.write_text(lib_names_text)
         logger.info(f"Generated requirements file: {file}")
 
+    def _write_module_test(self) -> None:
+        """
+        Write out test scaffolding for refactored code in module file.
+        The scaffolding contains placeholders for testing each function
+        in the module file and is meant to be fleshed out by the user
+        to suit their needs. When run out of the box, it simply tests
+        whether each function in the module runs without error.
+        """
+        # Format components to be passed into file template
+        module_name = f"{self.pipeline_name}_module"
+        test_class_name = f"Test{self.pipeline_name.title().replace('_', '')}"
+        function_metadata_list = [
+            {
+                "function_name": f"get_{node_collection.safename}",
+                "function_arg_names": sorted(
+                    [v for v in node_collection.input_variables]
+                ),
+                "test_function_name": f"test_get_{node_collection.safename}",
```

Comment:
This can be skipped. add the "test_" prefix directly into the template.

Reply:
Done